pvlib/tests/test_solarposition.py

```diff
@@ -786,8 +787,12 @@ def test_get_solarposition_microsecond_index(method, tz):
     index_ns = pd.date_range(unit='ns', **kwargs)
     index_us = pd.date_range(unit='us', **kwargs)
 
-    sp_ns = solarposition.get_solarposition(index_ns, 40, -80, method=method)
-    sp_us = solarposition.get_solarposition(index_us, 40, -80, method=method)
+    with warnings.catch_warnings():
+        # don't warn on method reload
+        warnings.simplefilter("ignore")
```

Comment:
This change is not directly related; just a tweak to suppress nuisance reloading warnings